pkg/apiserver/rest/dualwriter_mode1.go

```diff
@@ -274,6 +275,11 @@ func (d *DualWriterMode1) Destroy() {
 	d.Legacy.Destroy()
 }
 
+func (d *DualWriterMode1) Watch(ctx context.Context, options *metainternalversion.ListOptions) (watch.Interface, error) {
+	klog.Error("Watch not implemented in mode 1")
```

Comment:
```suggestion
	d.Log.Error("Watch not implemented in mode 1")
```

Reply:
By using the `d.Log` structure we also get the `DualWriterMode1` label, which helps when we want to filter logs by mode.

Also, I think it would be ok to have an error like `ErrNotImplemented`